pkg/cmd/roachtest/acceptance.go

```diff
@@ -58,7 +58,6 @@ func registerAcceptance(r *testRegistry) {
 			// to head after 19.2 fails.
 			minVersion: "v19.2.0",
 			timeout:    30 * time.Minute,
-			skip:       "https://github.com/cockroachdb/cockroach/issues/64491",
```

Comment:
Not sure if this line should be reverted?

Reply:
Yup, it should.